scripts/ci_setup.py

```diff
@@ -309,7 +317,11 @@ def __main(args: list) -> int:
         target_framework_moniker = dotnet.FrameworkAction.get_target_framework_moniker(framework)
         dotnet_version = dotnet.get_dotnet_version(target_framework_moniker, args.cli) if args.dotnet_versions == [] else args.dotnet_versions[0]
         commit_sha = dotnet.get_dotnet_sdk(target_framework_moniker, args.cli) if args.commit_sha is None else args.commit_sha
-        source_timestamp = dotnet.get_commit_date(target_framework_moniker, commit_sha, repo_url)
+        if(args.commit_time is not None):
```

Comment:
Should we instead fallback to grabbing the date from Github if we see a parse error? Otherwise, I don't really see a strong need to keep around the other code path.

Reply:
Yes, I think if the commit_time is not included or fails to parse that is the correct approach.